lib/src/components/dropdown/dropdown.dart

```diff
@@ -55,9 +55,9 @@ class ZetaDropdown<T> extends StatefulWidget {
   const ZetaDropdown({
     required this.items,
     this.onChange,
+    @Deprecated('disabled is deprecated. To disable the dropdown, set onChange to null') bool disabled = false,
```

Comment:
(Optional)

I am unsure what best practice for deprecation warnings is, but I have been copying the format flutter internal uses, which would make this:
```dart
@Deprecated('Set onChange to null. ' 'Disabled is deprecated as of 0.11.0')
````
I am not sure why it is in 2 strings spaced apart, but it works.